tree/dataframe/inc/ROOT/RDF/ActionHelpers.hxx

```diff
@@ -61,6 +63,13 @@ public:
    template <typename... Args>
    void CallFinalizeTask(unsigned int, Args...) {}
 
+   // Helper functions for RMergeableValue
```

Comment:
please remove this comment or move it below: the virtual dtor is not an helper function for RMergeableValue, it's a bug fix (thanks, by the way :smile: )